src/main/java/calebyyy/Main.java

```diff
@@ -0,0 +1,31 @@
+package calebyyy;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private Calebyyy duke = new Calebyyy();
```

Comment:
Can consider renaming the remnants of duke for decreased confusion in the long term!